pcl_ros/include/pcl_ros/pcl_node.hpp

```diff
@@ -113,23 +170,23 @@ class PCLNodelet : public nodelet_topic_tools::NodeletLazy
    * specified jitter. See also @ref latched_indices_ and approximate_sync.
    **/
   bool use_indices_;
-  /** \brief Set to true if the indices topic is latched.
+  /** \brief Set to true if the indices topic has transient_local durability.
    *
    * If use_indices_ is true, the ~input and ~indices topics generally must
    * be synchronised in time. By setting this flag to true, the most recent
    * value from ~indices can be used instead of requiring a synchronised
    * message.
    **/
-  bool latched_indices_;
+  bool latched_indices_;  // TODO(daisukes): should this be renamed to like `transient_local_indices_`?
```

Comment:
Yeah I think that would be wise. There will come a time when ROS 1 terminology will be antiquated